### PR TITLE
Mark blocked contacts / disable sending messages to blocked contacts

### LINKED ISSUE #135: Blocked contacts can still be contacted
## Expected Behavior

In the app, conversations with blocked contacts don't show a text entry field anymore. Threema Web should probably copy that behavior.

## Current Behavior

Blocked contacts can still be contacted.

## Possible Solution

- Disable input field
- Update placeholder text

Not sure if this needs an app update.

### PR DESCRIPTION
**Pull request by sillych**
Fix #135

Avatar with block contact icon:
2017-09-07-152352_324x80_scrot

Similar to the contact detail view in the android app, blocked status is shown as text:
2017-09-07-152358_845x200_scrot

Sending messages to a blocked contact shows a error message:
2017-09-07-152422_847x144_scrot

### CODE REVIEW

File: public/i18n/en.json [hunk 1]
```diff
@@ -175,6 +176,7 @@
         "MESSAGE_NOT_SUPPORTED": "«{receiverName}» cannot receive this message.",
         "AUDIO_MESSAGES_NOT_SUPPORTED": "«{receiverName}» cannot receive voice messages.",
         "FILE_MESSAGES_NOT_SUPPORTED": "«{receiverName}» cannot receive files.",
+        "RECEIVER_BLOCKED":  "You cannot send messages to a blocked receiver",
```
[the remark this answers is not in the document]
Comment (sillych) [reply public/i18n/en.json]: "Receiver" is more generic, e.g. if we support blocking Group Chats etc...

[the remark this answers is not in the document]
Comment (ovalseven8) [reply public/i18n/en.json]: > But I wonder whether we should disable the input field for blocked contacts.

Sounds good.

File: public/i18n/de.json [hunk 2]
```diff
@@ -173,6 +174,7 @@
         "MESSAGE_NOT_SUPPORTED": "«{receiverName}» kann diese Nachricht noch nicht erhalten.",
         "AUDIO_MESSAGES_NOT_SUPPORTED": "«{receiverName}» kann noch keine Sprachnachrichten erhalten.",
         "FILE_MESSAGES_NOT_SUPPORTED": "«{receiverName}» kann noch keine Dateien erhalten.",
+        "RECEIVER_BLOCKED":  "Sie können keine Nachrichten an blockierte Empfänger senden",
```
Comment (ovalseven8) [inline public/i18n/de.json]: Dot at the end of sentence.

File: public/i18n/de.json [hunk 3]
```diff
@@ -173,6 +174,7 @@
...
         "ERROR_OCCURRED": "Es ist ein Fehler aufgetreten.",
         "FILE_TOO_LARGE": "Aktuell können keine Dateien grösser als 15 MiB über Threema Web versendet werden",
```
Comment (ovalseven8) [inline public/i18n/de.json]: Dot at the end of sentence.

**Pull request merged**